docs/datagrid/filters.md

```diff
@@ -0,0 +1,48 @@
+Column filters in the Datagrid
+===
+The DataGrid allows you to set the filtering option per column. The basic one is sorting.
+
```

Comment:
Are we not mixing concepts here? For me, filtering and sorting are two different things. 🤔

Reply:
Sorting is kind of filtering, I wouldn't like to create so many articles about similar things.

Reply:
Filtering assumes that we filter out some data without changing its order, both can be described more as... 'Querying'? Or data organizing? Data processing?

Reply:
@stitot any suggestion?

Reply:
From a technical point of view they might be the same/similar, but behaviorally they are quite different. I believe we should distinguish quite clearly between `sorting` and `filtering` in docs and API. Two different concepts.

Reply:
So, should the sorting be a separate article or a part of Column article?